src/features/components/templates/banners/trade-type-banner/data.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+
+export const banner_images = {
+    banner: (
+        <StaticImage
+            src="../../../../../images/common/trade-types/trade-patterns.png"
+            alt="Trade type banner"
+            formats={['avif', 'webp', 'auto']}
+            placeholder="none"
+            style={{ height: '45rem' }}
```

Comment:
Can you try height prop instead of this?

Reply:
Since, it will be in the top of page so we should use `loading="eager"` here. This will help us reduce LCP

Reply:
I'm setting height for the image container. I tried height props, it doesn't help in this case.